js/main.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+const api_URL = 'https://opentdb.com/api.php?amount=10&type=multiple';
+const questionNum = document.getElementById('questionNum');
+const question = document.getElementById('question');
+const choices = document.getElementById('choices');
+const btn = document.getElementById('btn');
+const category = document.getElementById('category');
+const difficulty = document.getElementById('difficulty');
+let quizSet = [];
+let currentNum = 0;
+let score = 0;
+// クイズ取得
+const fetchQuizData = () => {
```

Comment:
呼び出し元は `async` をつけます

```suggestion
const fetchQuizData = async () => {
```